### ISSUE TITLE
cannot start up docker container

### ISSUE THREAD

**Issue by BasicWalker** [cmdr2/stable-diffusion-ui#28]
build was successful
using windows 10, docker-compose version 1.29.2, build 5becea4c

after running `docker-compose up`
```
Starting sd ... error

ERROR: for sd  Cannot start service stability-ai: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: signal: segmentation fault, stdout: , stderr:: unknown

ERROR: for stability-ai  Cannot start service stability-ai: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: signal: segmentation fault, stdout: , stderr:: unknown
ERROR: Encountered errors while bringing up the project.
```

**Comment by arkilis** [comment]
Got the same error.

**Comment by UrielCh** [comment]
Please give mode detail.

**Comment by cmdr2** [comment]
Hi @BasicWalker @arkilis  Are you using this with [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)?

@arkilis Which OS version are you using?

Thanks

**Comment by BasicWalker** [comment]
> Hi @BasicWalker @arkilis Are you using this with [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)?
> 
> @arkilis Which OS version are you using?
> 
> Thanks

Most recent version of wsl running Ubuntu 22.04

**Comment by arkilis** [comment]
@BasicWalker Have you sorted out this issue? Thx

**Comment by cmdr2** [comment]
Hi @arkilis and @BasicWalker 

Can you please try these steps:
1. `wsl --shutdown` and then `wsl --update`
2. Please ensure that you have nvidia-container-toolkit installed: `sudo apt-get update -y` and then `sudo apt-get install -y nvidia-container-toolkit` then please restart your docker service.
3. If all these fail, you can try the v2 beta version of this project, which doesn't require docker or WSL. Just install and run: https://github.com/cmdr2/stable-diffusion-ui/tree/v2#system-requirements

Thanks

**Comment by cmdr2** [comment]
@BasicWalker Yes, everything is within the WSL console

**Comment by cmdr2** [comment]
@BasicWalker @arkilis Can you please try the v2 version instead? It does not require docker or WSL anymore. You can just download one file, and it'll install everything behind-the-scenes and start.

You can download it from https://drive.google.com/file/d/1cEuOcb9OaldXcc2XzEMqEdvAr_w-KJ_p/view?usp=sharing and then double-click `Start Stable Diffusion UI.cmd` to install and run.

Please let me know if there are any problems after running it.

PS: There's a new Discord server for support and development discussions: https://discord.com/invite/u9yhsFmEkB . Please join in for faster discussion and feedback on v2. Thanks!

**Comment by arkilis** [comment]
@cmdr2 Thanks for your support! Works now!!

**Comment by ThierryBegin** [comment]
I have the same issue on Linux Mint. V2 seem to be windows only?

**Comment by cmdr2** [comment]
@ThierryBegin A Linux version of v2 is nearly ready. I'll update here once you can try it. Thanks!

In the meanwhile, can you please try these steps? https://github.com/cmdr2/stable-diffusion-ui/issues/28#issuecomment-1233743029

**Comment by ThierryBegin** [comment]
@cmdr2 unfortunately im at the latest available and cuda is still at 11.4, I'll have to wait for V2, Thanks!

**Comment by cmdr2** [comment]
Hi @ThierryBegin - a linux native version is available now (in beta)! https://github.com/cmdr2/stable-diffusion-ui/blob/v2/README.md#installation

Please let me know if this works (or doesn't), thanks! :)
